### ISSUE TITLE
Flaky spec: Booth Officer with single booth assignment today

### ISSUE THREAD

**Issue by bertocq** [AyuntamientoMadrid/consul#1204]
Warning!
===
This spec probably only fails between 00:00 and 02:00 am Madrid time

# What
Tests that fail randomly are called "flakies", this one seems to be one:

**Randomized seed:** 19087

**Failure:**

```
    3) Booth Officer with single booth assignment today
     Failure/Error:
       within("#officing-booth") do
         expect(page).to have_content "You are officing the booth located at #{booth.location}."
       end
     Capybara::ElementNotFound:
       Unable to find visible css "#officing-booth"
     # ./spec/features/officing/booth_spec.rb:33:in `block (2 levels) in <top (required)>'
```

# How
- [ ] Explain why the test is flaky, or under which conditions/scenario it fails randomly
- [ ] Explain why your PR fixes it
- [ ] Create a backport PR to consul/consul when the fixing PR is approved

### Random values issues
If the problem comes from randomly generated values, running multiple times a single spec could help you reproduce the failure by running at your command line:
```bash
for run in {1..10}
do
  bin/rspec ./spec/features/budgets/investments_spec.rb:256
done
```

You can also try running a single spec in Travis:
Add option `:focus` to the spec and push your branch to Github, for example:
```ruby
scenario 'Show', :focus do
```

But remember to remove that `:focus` changes afterwards when submitting your PR changes!

### Test order issues
Running specs in the order they failed may discover that the problem is that a previous test sets an state in the test environment that makes our flaky fail/pass. Tests should be independent from the rest.

After executing rspec you can see the seed used, add it as an option to rspec, for example:
`bin/rspec --seed 55638` (check **Randomized seed** value at beginning of issue)

### Other things to watch for
- Time related issues (current time, two time or date comparisons with miliseconds/time when its not needed)
- https://semaphoreci.com/community/tutorials/how-to-deal-with-and-eliminate-flaky-tests

**Comment by bertocq** [comment]
Happened at https://travis-ci.org/AyuntamientoMadrid/consul/jobs/338257557

**Comment by javierm** [comment]
Closed via consul#2712 and #1625.
